src/stack.h

```diff
@@ -23,11 +23,36 @@ extern const std::array<const char*, LAYER_COUNT> g_layer_names;
 typedef enum SliceType {
     SLICE_CLIENT,
     SLICE_WINDOW,
-    SLICE_MONITOR,
 } SliceType;
 
 class Client;
-class Monitor;
+
+template<typename T>
+class PlainStack {
```

Comment:
Does this need to be in here? I'd prefer it getting its own `plainstack.h`.

Reply:
I wasn't sure, but I think it's reasonable to move it. So yes, it's now in `plainstack.h`.